extension/memorylimiterextension/metadata.yaml

```
@@ -0,0 +1,7 @@
+type: memory_limiter
+
+status:
+  class: extension
+  stability:
+    alpha: [extension]
+  distributions: [core, contrib]
```

It's not included in any distributions yet
```suggestion
  distributions: []
```

---

done